examples/plot_atlas.py

```diff
@@ -0,0 +1,16 @@
+"""
+Basic Atlas interaction
+=======================
+
+Plot the regions of a reference atlas.
+"""
+
+import matplotlib.pyplot as plt
+from nilearn.datasets import fetch_yeo_2011_atlas
+from nilearn import plotting
+
+atlas = fetch_yeo_2011_atlas()
+print atlas.DESCRIPTION
```

Comment:
After discussion with the group, we feel that it was better renaming this variable to lowercase.

I am implementing this change right now and merging. Thanks @bthirion